### PR TITLE
CB-42: Add user information to navbar

### PR DESCRIPTION
**Pull request by fortunee**
# JIRA TICKET NAME:
CB-42
## SUMMARY:
This PR basically gets the role information of the currently logged in user and displays it to the user.

### CODE REVIEW

File: app/js/components/common/Header.js [hunk 1]
```diff
@@ -108,7 +112,7 @@ export class Header extends Component {
 
                     <Link to="" activeClassName="active">
                         <li>
-                            <a href="#">Logout
+                            <a href="#">Logout {' '}
```
Comment (dkayiwa) [inline app/js/components/common/Header.js:115]: Are the empty brackets for decoration? 😊

Comment (fortunee) [reply app/js/components/common/Header.js:115]: Yes. It's just leaving a space between the icon and the text.

Comment (dkayiwa) [reply app/js/components/common/Header.js:115]: Isn't that what &nbsp is for?

Comment (fortunee) [reply app/js/components/common/Header.js:115]: That works, but it leaves a much wider space than it would like this.

**Pull request merged**